Gemfile

```diff
@@ -117,7 +126,7 @@ gem 'mini_magick'
 gem 'multi_xml'
 gem 'nokogiri'
 gem 'omniauth', '~> 1.3.1'
-gem 'rails', '~> 5.0.1'
+gem 'rails', '~> 5.0.2'
```

Comment:
Absolutely agree. Not sure this will end up being the final PR as is, might refactor the commit history or something in the end. I tend to test things as I go and commit in small chunks. Bumped this one (i think) due to a requirement in one of the other gems I bumped.